package-lock.json

```diff
@@ -1,141 +1,182 @@
 {
   "name": "bolt-ts-starter-template",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
```

Comment:
Sweeeeet 🚀  This is such a welcomed version bump!

We could also change the name here and in `package.json` to `bolt-ts-custom-function-template` IMO but it's no blocker at all